src/components/Project/Project.module.scss

```diff
@@ -0,0 +1,64 @@
+@import '../../assets/styles/theme';
+.mobileProjectTitle {
+  color: $white;
+  font: 100% $helvetica;
+  font-size: 32px;
+  text-align: center;
+  margin: 80px 0px 32px 0px;
+}
+.mobileProject {
+  font: 100% $helvetica;
+  text-align: center;
+  background-color: $black_100_35;
+  color: $white;
+  margin: 0px 8px;
+  font-size: 16px;
+  padding: 32px 24px 50px 24px;
+
+  div {
+    text-align: justify;
+    padding-bottom: 50px;
+  }
+
+  a:hover,
+  a:link,
+  a:visited,
+  a:active {
+    font-size: 14px;
+    font-weight: 600;
+    text-decoration-line: none;
+  }
+}
+
+//dark theme
+
+.mobileProjectTitle {
+  color: $white;
+}
+.mobileProject {
+  background-color: $black_100_35;
+  color: $white;
+
+  a:hover,
+  a:link,
+  a:visited,
+  a:active {
+    color: $white;
+  }
+}
+//light theme
+
+.mobileProjectTitle {
+  color: $black;
+}
```

Comment:
currently light theme is overriding dark theme, this is not the best solution, however, as we do not have theme switching implemented yet, I think its okay to leave it like that for now, we'll change it once we have the switiching mechanism